Source/Platform/DefaultOpenTK/Backend/Graphics/NativeWindow.cs

```diff
@@ -51,14 +49,17 @@ public int Width
 		{
 			get { return this.internalWindow.ClientSize.Width; }
 		}
+
```

Comment:
Please remove newlines between properties to keep in line with the style of the codebase.